packages/jest-cli/src/reporters/coverage_reporter.js

```diff
@@ -83,8 +83,14 @@ export default class CoverageReporter extends BaseReporter {
     await this._addUntestedFiles(this._globalConfig, contexts);
     let map = this._coverageMap;
     let sourceFinder: Object;
-    if (this._globalConfig.mapCoverage) {
-      ({map, sourceFinder} = this._sourceMapStore.transformCoverage(map));
+    const transformedCoverage = this._sourceMapStore.transformCoverage(map);
```

Comment:
might actually be better to keep `mapCoverage` here? I don't really understand why it's necessary 🙁